Makefile

```diff
@@ -40,9 +40,15 @@ show-docs:
 
 linkcheck:
 	uv pip install -e ".[docs]"
+	# this checks: mkdocs.yml is valid, all listed pages exist, plugins are correctly configured, no broken references in nav or Markdown links (internal), broken links and images (internal, not external)
 	mkdocs build --strict
-	vale docs
-	linkchecker site/
+	# lychee checks for broken external links in the built site
+	lychee --offline site/
```

Comment:
Actually I think this needs to call `mkdocs serve` also so the `site/` is created for `lychee` to check the links.

Reply:
`mkdocs serve` doesn’t actually generate the site/ directory, I don’t think it does. It’s `mkdocs build` that creates the site/ folder.

What made you think otherwise? Did you run into any issues?" @ankatiyar ?